panel/pane/perspective.py

```diff
@@ -382,10 +379,6 @@ def _get_properties(self, doc, source=None):
         if 'theme' in props and 'material' in props['theme']:
             props['theme'] = props['theme'].replace('material', 'pro')
         del props['object']
-        if props.get('toggle_config'):
```

Comment:
I'm in double if this should be replaces with 'settings' check instead of removed?

Reply:
I don't think this is needed anymore. I think it's a leftover from before 1.0 when explicit sizing was required.

Reply:
Turns out this was needed 😄